build.gradle

```diff
@@ -305,7 +307,7 @@ configure(javaProjects) {
     task shadedJar(type: ShadowJar,
                    group: 'Build',
                    description: 'Builds the shaded JAR.',
-                   dependsOn: tasks.classes) {
+                   dependsOn: [ tasks.classes, tasks.processResources ]) {
```

Comment:
According to this document, classes depends on processResources so this should be a no-op. Are you seeing a change with it?

https://docs.gradle.org/current/userguide/java_plugin.html